src/ngraph/file_util.cpp

```diff
@@ -186,15 +186,14 @@ string file_util::get_temp_directory_path()
 {
     const vector<string> potential_tmps = {"NGRAPH_TMP", "TMPDIR", "TMP", "TEMP", "TEMPDIR"};
 
-    string path;
-    for (const string& var : potential_tmps)
+    string path = getenv_string(ngraph::EnvVarEnum::NGRAPH_TMPDIR);
+    /*for (const string& var : potential_tmps)
```

Comment:
Don't use multi-line comments
Remove the code or leave it.
Why was this removed? It is useful.

Reply:
I added a new environment variable `NGRAPH_TMPDIR` to specify temp directory and removed the ambiguity of potential temp values.